randovania/games/dread/json_data/Ferenia.txt

```diff
@@ -3161,13 +3163,15 @@ Extra - asset_id: collision_camera_040
   > Dock to Purple EMMI Introduction Access
       Trivial
   > Tile Group (POWERBOMB)
-      Space Jump or Infinite Bomb Jump (Beginner)
+      Any of the following:
+          Space Jump or Simple IBJ
+          Flash Shift and Speed Booster
   > Tile Group (WEIGHT) 1
       Can Slide
   > Tunnel to EMMI Zone Exit West
-      Any of the following:
-          Simple IBJ or Use Spin Boost
-          Flash Shift and Walljump (Beginner)
+      All of the following:
+          After Ferenia - Central Unit
+          Flash Shift or Spin Boost or Simple IBJ
```

Comment:
Remove the CU event; it's covered by the dock weakness. Can add the Flash/Cross + speed for here too